static/app/utils/metrics/types.tsx

```diff
@@ -100,7 +100,7 @@ export type MetricCorrelation = {
   }[];
 };
 
-export type MetricRange = {
+export type SelectionRange = {
```

Comment:
Hey, what do you think of changing types to interfaces, whenever we touch them?

Reply:
Agreed :D